CRITICAL - 2024-10-07 15:41:56 --> Error: Call to undefined function App\Controllers\UsuarioModel()
[Method: GET, Route: usuarios]
in APPPATH\Controllers\UsuarioController.php on line 12.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsuarioController->index()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuarioController))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-10-07 15:42:14 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\laragon\www\fotobook\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\fotobook\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\laragon\www\fotobook\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\fotobook\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\fotobook\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\fotobook\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\fotobook\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\fotobook\app\Controllers\UsuarioController.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\fotobook\system\CodeIgniter.php(933): App\Controllers\UsuarioController->index()
#8 C:\laragon\www\fotobook\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuarioController))
#9 C:\laragon\www\fotobook\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\fotobook\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\fotobook\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\fotobook\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\laragon\www\fotobook\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\fotobook\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\fotobook\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\fotobook\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\fotobook\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\fotobook\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\fotobook\app\Controllers\UsuarioController.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\fotobook\system\CodeIgniter.php(933): App\Controllers\UsuarioController->index()
#7 C:\laragon\www\fotobook\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuarioController))
#8 C:\laragon\www\fotobook\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\fotobook\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\fotobook\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\fotobook\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-10-07 15:42:14 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
[Method: GET, Route: usuarios]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios_225025`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\UsuarioController.php(14): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsuarioController->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuarioController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-10-07 15:44:27 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for arturozaisa.com failed: Host desconocido.  in C:\laragon\www\fotobook\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\laragon\\www\\...', 187)
#1 C:\laragon\www\fotobook\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('arturozaisa.com', 'ugsjwgu5z7wgt', 'guarache', 'dbhggkubvmugoy', 3306, '', 0)
#2 C:\laragon\www\fotobook\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\laragon\www\fotobook\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\laragon\www\fotobook\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\laragon\www\fotobook\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#6 C:\laragon\www\fotobook\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\laragon\www\fotobook\app\Controllers\UsuarioController.php(14): CodeIgniter\BaseModel->findAll()
#8 C:\laragon\www\fotobook\system\CodeIgniter.php(933): App\Controllers\UsuarioController->index()
#9 C:\laragon\www\fotobook\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuarioController))
#10 C:\laragon\www\fotobook\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\fotobook\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\laragon\www\fotobook\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\laragon\www\fotobook\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for arturozaisa.com failed: Host desconocido.  in C:\laragon\www\fotobook\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\fotobook\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('arturozaisa.com', 'ugsjwgu5z7wgt', 'guarache', 'dbhggkubvmugoy', 3306, '', 0)
#1 C:\laragon\www\fotobook\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\fotobook\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\fotobook\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\fotobook\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\fotobook\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\fotobook\app\Controllers\UsuarioController.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\fotobook\system\CodeIgniter.php(933): App\Controllers\UsuarioController->index()
#8 C:\laragon\www\fotobook\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuarioController))
#9 C:\laragon\www\fotobook\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\fotobook\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\fotobook\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\fotobook\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for arturozaisa.com failed: Host desconocido.  in C:\laragon\www\fotobook\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\fotobook\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\fotobook\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\fotobook\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\fotobook\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\fotobook\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\fotobook\app\Controllers\UsuarioController.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\fotobook\system\CodeIgniter.php(933): App\Controllers\UsuarioController->index()
#7 C:\laragon\www\fotobook\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuarioController))
#8 C:\laragon\www\fotobook\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\fotobook\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\fotobook\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\fotobook\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-10-07 15:44:27 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for arturozaisa.com failed: Host desconocido. 
[Method: GET, Route: usuarios]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios_225025`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\UsuarioController.php(14): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\UsuarioController->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuarioController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
